src/components/Footer/index.tsx

```diff
@@ -20,80 +18,70 @@ interface FooterProps {
   /** The name of the license of the project. */
   license: License
   /** A list of the developers who created the site. */
-  developers: string | Array<string>
+  developers: string | string[]
   /** The year that the story was published, e.g., 2018. */
   copyrightYear: number
-  /** What type of project this footer is being used in. */
-  footerType: ComponentType
   /** Custom css for the footer component */
   style?: string
 }
 
 /** A footer to go at the bottom of every page. */
 class Footer extends React.Component<FooterProps> {
-  static defaultProps = {
+  public static defaultProps = {
     license: License.Copyright,
-    footerType: ComponentType.Default,
   }
 
-  render() {
-    if (this.props.footerType === ComponentType.Default) {
-      return (
-        <footer
-          className={css`
-            display: grid;
-            grid-template-columns: 1fr;
-            grid-template-rows: auto auto;
-            justify-content: center;
-            margin: 1rem 0;
-            padding: 0 0.4rem;
-            text-align: center;
+  public render() {
```

Comment:
curious why public render instead of render, quick lookup can't find an answer.

Reply:
We should make a list somewhere and have a discussion about tslint rules that we want to change?

Reply:
^^ yes. I was thinking I'd like to make a `tslint-config-dailybruin` package so that we can have one unified ruleset across all our projects. Let's talk about rules sometime this week?